swig/cpp/examples/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}")
+
+add_executable(get_item_example get_item_example.cpp)
+add_executable(set_item_example set_item_example.cpp)
+add_executable(delete_item_example delete_item_example.cpp)
+
+target_link_libraries(get_item_example Sysrepo_cpp)
+target_link_libraries(set_item_example Sysrepo_cpp)
+target_link_libraries(delete_item_example Sysrepo_cpp)
+
+macro(INSTALL_EXAMPLE_YANG MODULE_NAME REVISION)
```

Comment:
There is no need to install the YANG models and data again, since these are already installed once by the rules in /exeamples/ directory if examples are enabled.